Automattic/wp-calypso
# Earn: Convert membership constants to TS

## Proposed Changes

Converts Earn > Memberships > Constants file to TS. In this case, just renames file. Part of broader effort to update all Earn code. 

## Testing Instructions

Load this branch, go to the Earn > Memberships page at `http://calypso.localhost:3000/earn/payments/YOURDOMAIN`, and test various things to confirm the look and behavior on this page is the same as it was before. 

## Pre-merge Checklist
